playbook/app/pb_kits/playbook/pb_text_input/text_input.rb

```diff
@@ -41,6 +43,10 @@ def validation_data
       def error_class
         error ? " error" : ""
       end
+
+      def inline_class
+        inline ? " inline" : ""
```

Comment:
`inline ? "inline" : null`
You don't need the space I think if you use null instead of an empty string.